events/code_review_events/workflow.py

```diff
@@ -219,7 +225,44 @@ def publish_results(self, payload):
         return True
 
     def parse_pulse(self, payload):
-        assert self.publish is True, "Publication disabled"
+
+        routing = payload["routing"]
+
+        # Process autoland payloads
+        if routing["exchange"] == PULSE_TASK_GROUP_RESOLVED:
+            try:
+                self.trigger_autoland(payload["body"])
+            except Exception as e:
+                logger.warn(
+                    "Autoland trigger failure", key=routing["key"], error=str(e)
+                )
+        else:
+            logger.info("Skipping pulse message", key=routing["key"])
```

Comment:
This will flood the log with messages, as we have a lot of task completed messages.